src/bin/main.rs

```diff
@@ -9,6 +10,7 @@ fn main() -> Result<()> {
             bevy_cli::template::generate_template(&new.name, &new.template, &new.branch)?;
         }
         Subcommands::Lint { args } => bevy_cli::lint::lint(args)?,
+        Subcommands::Build(args) => build(&args)?,
```

Comment:
```suggestion
        Subcommands::Build(args) => bevy_cli::build::build(&args)?,
```

(Super small nit) The other subcommands use the full qualified path. I personally think it makes it more clear that the function is external, but I won't push it.